app/models/family.rb

```diff
@@ -86,6 +86,10 @@ def everyone_has_birthdates?
   def anyone_has_email?
     attendees.any? { |p| p.email.present? }
   end
+ 
+  def anyone_registered_for_legacy_conferences()
+    attendees.find{|a| a.conferences.find{ |c| c.name == 'Legacy'}} != nil
```

Comment:
Rubocop might consider the `!= nil` to be redundant, since the result is truthy/falsey without it.

Reply:
Yah rubocop wont run on this project right now either, obsolete configuration, another thing to try to get updated.  I think id rather leave it,  I want it to return true or false, and not a truthy value, as it will return the found attendees conference if it finds one which is not the intended use of the method.  Let me know what you think.

Reply:
That's fine.